server/app/services/applicant/question/TextQuestion.java

```diff
@@ -80,4 +82,14 @@ public Path getTextPath() {
   public String getAnswerString() {
     return getTextValue().orElse("-");
   }
+
+  @Override
+  public ImmutableMap<Path, String> getJsonEntries() {
+    return applicantQuestion.getContextualizedScalars().keySet().stream()
+        .filter(path -> !Scalar.getMetadataScalarKeys().contains(path.keyName()))
+        .collect(
+            toImmutableMap(
+                path -> path,
+                path -> applicantQuestion.getApplicantData().readAsString(path).orElse("")));
+  }
```

Comment:
Above code is taken from the default case in:

https://github.com/civiform/civiform/blob/358ad3dbb6e67aacb91afb3ede60e32ac281ba26/server/app/services/applicant/ReadOnlyApplicantProgramServiceImpl.java#L450-L508